src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -5282,6 +5282,56 @@ contextState switch
                 };
         }
 
+        protected override IParameterSymbol GetCorrespondingParameterCore(SyntaxNode node, CancellationToken cancellationToken)
```

Comment:
>SyntaxNode [](start = 74, length = 10)

I think we would want the actual work to be done in a method that is strongly typed.